lib/bundler/plugin/installer.rb

```diff
@@ -38,22 +43,29 @@ def definition.lock(*); end
 
     private
 
+      # Rubocop misunderstands the semantics of this method, assuming an `else` code block
+      # that doesn't exist. See https://github.com/bbatsov/rubocop/issues/5702.
+      #
+      # rubocop:disable Style/GuardClause
+      def check_sources_consistency!(options)
+        if options[:git] && options[:'local-git']
+          raise InvalidOption, "Remote and local plugin git sources can't be both specified"
+        end
+      end
+      # rubocop:enable Style/GuardClause
+
       def install_git(names, version, options)
         uri = options.delete(:git)
         options["uri"] = uri
 
-        source_list = SourceList.new
-        source_list.add_git_source(options)
-
-        # To support both sources
-        if options[:source]
-          source_list.add_rubygems_source("remotes" => options[:source])
-        end
+        install_all_sources(names, version, options, options[:source])
+      end
 
-        deps = names.map {|name| Dependency.new name, version }
+      def install_local_git(names, version, options)
+        uri = options.delete(:'local-git')
+        options["uri"] = uri
```

Comment:
I like to not mix the "case" of my hashes - less surprise to have them all the same. This change is for the downstream code on l87, correct?

Reply:
(I was thinking "oh just use `stringify_keys`" but that's a Rails thing.. 🙄 )

Reply:
I think this can't be changed in isolation :scream:; following the code path, it leads to https://github.com/bundler/bundler/blob/77dbd12d0759ca635e3cb5cb0e90840cdac7f0d0/lib/bundler/source/git.rb#L23

Reply:
Ah, ok.. well, shoot, that's a pain. It might behoove us to do some work to clean up inconsistencies in the codebase as a whole.. but for now, 👍